### PR TITLE
Enhancement: Document a new section for popular command line frameworks

### PR DESCRIPTION
**Pull request by UrielOfir**
I tried to solve issue #78.
I would be glad to get feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -923,6 +925,57 @@ Guidelines:
 
 References for other CLIs which collect analytics are [Angular CLI](https://angular.io/analytics), and [Next.js](https://nextjs.org/telemetry) CLI.
 
+# 9 Appendix: CLI Frameworks
+
+### 9.1 CLI Frameworks Table
+
+<!-- markdownlint-disable -->
+<table>
```
Comment (lirantal) [inline README.md]: This is great but can you convert the HTML table into Markdown syntax for tables? See here: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables

**Review by lirantal — CHANGES_REQUESTED** [review]

**Comment by lirantal** [comment]
Thanks for taking this on @UrielOfir 👏
I added one review item to fix and I'll merge it right after

File: README.md [hunk 2]
```diff
@@ -929,51 +929,17 @@ References for other CLIs which collect analytics are [Angular CLI](https://angu
 
 ### 9.1 CLI Frameworks Table
 
+| Name  | Description | Npm | Github | Stars and downloads |
```
Comment (lirantal) [inline README.md]:
```suggestion
| Name  | Description | npm | GitHub | Stars and downloads |
```

File: README.md [hunk 3]
```diff
@@ -929,51 +929,17 @@ References for other CLIs which collect analytics are [Angular CLI](https://angu
...
+| ------------- | ------------- | ------------- | ------------- | ------------- |
+| oclif  | A framework for building a command line interface.  | [Link to npm](https://www.npmjs.com/package/oclif) | [Link to Github](https://github.com/oclif/oclif) | ![](https://img.shields.io/github/stars/oclif/oclif)![](https://img.shields.io/npm/dt/oclif.svg)
```
Comment (lirantal) [inline README.md]:
```suggestion
| oclif  | A framework for building a command line interface.  | [Link to npm](https://www.npmjs.com/package/oclif) | [Link to GitHub](https://github.com/oclif/oclif) | ![](https://img.shields.io/github/stars/oclif/oclif)![](https://img.shields.io/npm/dt/oclif.svg)
```

File: README.md [hunk 4]
```diff
@@ -929,51 +929,17 @@ References for other CLIs which collect analytics are [Angular CLI](https://angu
...
+| inquirer  | A collection of common interactive command line user interfaces.  | [Link to npm](https://www.npmjs.com/package/inquirer) | [Link to Github](https://github.com/SBoudrias/Inquirer.js) |![](https://img.shields.io/github/stars/sboudrias/inquirer.js)![](https://img.shields.io/npm/dt/inquirer.svg)
```
Comment (lirantal) [inline README.md]:
```suggestion
| inquirer  | A collection of common interactive command line user interfaces.  | [Link to npm](https://www.npmjs.com/package/inquirer) | [Link to GitHub](https://github.com/SBoudrias/Inquirer.js) |![](https://img.shields.io/github/stars/sboudrias/inquirer.js)![](https://img.shields.io/npm/dt/inquirer.svg)
```

File: README.md [hunk 5]
```diff
@@ -929,51 +929,17 @@ References for other CLIs which collect analytics are [Angular CLI](https://angu
...
+| ink | Ink provides the same component-based UI building experience that React offers in the browser, but for command-line apps. | [Link to npm](https://www.npmjs.com/package/ink) | [Link to Github](https://github.com/vadimdemedes/ink) | ![](https://img.shields.io/github/stars/vadimdemedes/ink)![](https://img.shields.io/npm/dt/ink.svg)
```
Comment (lirantal) [inline README.md:936]:
```suggestion
| ink | Ink provides the same component-based UI building experience that React offers in the browser, but for command-line apps. | [Link to npm](https://www.npmjs.com/package/ink) | [Link to GitHub](https://github.com/vadimdemedes/ink) | ![](https://img.shields.io/github/stars/vadimdemedes/ink)![](https://img.shields.io/npm/dt/ink.svg)
```

File: README.md [hunk 6]
```diff
@@ -929,51 +929,17 @@ References for other CLIs which collect analytics are [Angular CLI](https://angu
...
+|blessed |A curses-like library with a high level terminal interface API for node.js.| [Link to npm](https://www.npmjs.com/package/blessed) | [Link to Github](https://github.com/chjj/blessed) | ![](https://img.shields.io/github/stars/chjj/blessed)![](https://img.shields.io/npm/dt/blessed.svg)
```
Comment (lirantal) [inline README.md]:
```suggestion
|blessed |A curses-like library with a high level terminal interface API for node.js.| [Link to npm](https://www.npmjs.com/package/blessed) | [Link to GitHub](https://github.com/chjj/blessed) | ![](https://img.shields.io/github/stars/chjj/blessed)![](https://img.shields.io/npm/dt/blessed.svg)
```

**Review by lirantal — CHANGES_REQUESTED** [review]

**Comment by lirantal** [comment]
Looks wonderful @UrielOfir thank you ❤️ 
I added a bunch of small suggestions due to proper naming conventions and we're good to merge!

**Comment by UrielOfir** [comment]
> Looks wonderful @UrielOfir thank you ❤️ I added a bunch of small suggestions due to proper naming conventions and we're good to merge!

I approved your changes :)

**Review by lirantal — APPROVED** [review]

**Comment by lirantal** [comment]
@UrielOfir do you want to update the Teams section on the readme by adding your contributor card to it too? https://github.com/lirantal/nodejs-cli-apps-best-practices#the-team-

**Comment by UrielOfir** [comment]
> @UrielOfir do you want to update the Teams section on the readme by adding your contributor card to it too? https://github.com/lirantal/nodejs-cli-apps-best-practices#the-team-

I think I will wait until I'll contribute more.

**Comment by lirantal** [comment]
@UrielOfir any contribution, small or big, deserves to be celebrated and recognized. Please feel free to add yourself to the README ❤️

**Pull request merged**